npm-shrinkwrap.json

```diff
@@ -6,7 +6,7 @@
       "version": "1.0.3"
     },
     "abbrev": {
-      "version": "1.0.7"
+      "version": "1.0.9"
```

Comment:
So we lock down our top level dependencies, but sub-sub dependencies will update if for example nopt does not lock down their versions.
```
[kerryliu@gwwar ~/checkouts/wp-calypso] (master)$ npm ls abbrev
wp-calypso@0.17.0 /Users/kerryliu/checkouts/wp-calypso
├─┬ node-sass@3.4.2
│ └─┬ node-gyp@3.3.1
│   └─┬ nopt@3.0.6
│     └── abbrev@1.0.9 
└─┬ postcss-cli@2.5.1
  └─┬ chokidar@1.5.2
    └─┬ fsevents@1.0.12
      └─┬ node-pre-gyp@0.6.25
        └─┬ nopt@3.0.6
          └── abbrev@1.0.7 
```